two_factor_otp_auth/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 VentorTech OU
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0).
+
+{
+    "name": "Ventor Two Factor Authentication",
+    "summary": "Two factor authentication for users",
+    "author": "VentorTech OU",
```

Comment:
Author should be as in other merp modules "VentorTech"

Reply:
done